src/app/navBar/NavbarComponent.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+
+const NavbarComponent = () => {
+    return (
+        <nav className="navbarbg">
+            <div>
+                <div className="nav-wrapper" id="iconRight">
+                    <div className="Logo">
+                        <i className="far fa-play-circle" /> &nbsp;&nbsp;&nbsp;
+                        <a href="#!" className="brand-logo">
+                            LearnGround
+                        </a>
+                    </div>
+                    <div>
+                        <ul className="right hide-on-med-and-down">
+                            <li>
+                                <a href="#!">Science</a>
+                            </li>
+                            <li>
+                                <a href="#!">Technology</a>
+                            </li>
+                            <li>
+                                <a href="#!">Engineering</a>
+                            </li>
+                            <li>
+                                <a href="#!">Arts</a>
+                            </li>
+                            <li>
+                                <a href="#!">Mathematics</a>
+                            </li>
+                        </ul>
+                        <a
+                            href="#!"
+                            data-target="mobile-demo"
+                            className="sidenav-trigger"
+                        >
+                            <i className="material-icons">menu</i>
+                        </a>
+                    </div>
+                    <div className="hide-on-med-and-down show">
+                        <ul>
+                            <li>
+                                <a href="#!">SignIn</a>
+                            </li>
+                            <li style={{ color: '#5ed3f1' }}>|</li>
+                            <li>
+                                <a href="#!">SignUp</a>
+                            </li>
```

Comment:
Can you remove the inline styles?

Reply:
Thanks for spotting that.
I have removed the inline style